Dockerfile

```diff
@@ -1,81 +1,47 @@
-  echo "USING NODEJS VERSION <${NODEJS_VERSION}>" && \
-  echo "***********************************************************" && \
-  curl -s --fail http://nodejs.org/dist/v${NODEJS_VERSION}/node-v${NODEJS_VERSION}.tar.gz -o /opt/node-v${NODEJS_VERSION}.tar.gz && \
-  tar zxf /opt/node-v${NODEJS_VERSION}.tar.gz -C /opt && \
-  cd /opt/node-v${NODEJS_VERSION} && \
-  echo "INFO: Configure..." && ./configure && \
-  echo "INFO: Make..." && make -s V= && \
-  echo "INFO: Make install..." && make install && \
-  echo "INFO: node version <$(node -e "console.log(process.version)")>" && \
-  echo "INFO: npm version <$(npm --version)>" && \
+RUN \
+  apt-get update && \
+  apt-get install -y git && \
+  npm install pm2@3.2.2 -g && \
   echo "INFO: npm install --production..." && \
-  cd /opt/iota-lwm2m && npm install --production && \
-  echo "INFO: Cleaning unused software..." && \
-  yum erase -y gcc-c++ gcc ppl cpp glibc-devel glibc-headers kernel-headers libgomp libstdc++-devel mpfr libss yum-utils libxml2-python git && \
-  rm -rf /opt/node-v${NODEJS_VERSION}.tar.gz /opt/node-v${NODEJS_VERSION} && \
-  # Erase without dependencies of the document formatting system (man). This cannot be removed using yum
-  # as yum uses hard dependencies and doing so will uninstall essential packages
-  rpm -qa groff redhat-logos | xargs -r rpm -e --nodeps && \
-  # Clean yum data
-  yum clean all && rm -rf /var/lib/yum/yumdb && rm -rf /var/lib/yum/history && \
-  # Rebuild rpm data files
-  rpm -vv --rebuilddb && \
-  # Delete unused locales. Only preserve en_US and the locale aliases
-  find /usr/share/locale -mindepth 1 -maxdepth 1 ! -name 'en_US' ! -name 'locale.alias' | xargs -r rm -r && \
-  bash -c 'localedef --list-archive | grep -v -e "en_US" | xargs localedef --delete-from-archive' && \
-  # We use cp instead of mv as to refresh locale changes for ssh connections
-  # We use /bin/cp instead of cp to avoid any alias substitution, which in some cases has been problematic
-  /bin/cp -f /usr/lib/locale/locale-archive /usr/lib/locale/locale-archive.tmpl && \
-  build-locale-archive && \
-  find /opt/iota-lwm2m -name '.[^.]*' 2>/dev/null | xargs -r rm -rf && \
-  # Clean npm cache
-  npm cache clean && \
-  # Don't need unused files inside docker images
-  rm -rf /tmp/* /usr/local/lib/node_modules/npm/man /usr/local/lib/node_modules/npm/doc /usr/local/lib/node_modules/npm/html && \
-  # We don't need to manage Linux account passwords requisites: lenght, mays/mins, etc
-  # This cannot be removed using yum as yum uses hard dependencies and doing so will uninstall essential packages
-  rm -rf /usr/share/cracklib && \
-  # We don't need glibc locale data
-  # This cannot be removed using yum as yum uses hard dependencies and doing so will uninstall essential packages
-  rm -rf /usr/share/i18n /usr/{lib,lib64}/gconv && \
-  # Don't need old log files inside docker images
-  rm -f /var/log/*log
+  cd /opt/iotaul && npm install --production && \
```

Comment:
The `cd` both incorrect and  unnecessary.

If you want to keep it, it should be  `cd /opt/iota-lwm2m`, but the `WORKDIR` statement has already covered that.

I think consistency is the key here - either correct and leave in the redundant `cd` or remove it from the `Dockerfile` in the other Agents.

Reply:
Fixed in 281e5533cde77f8232f5633e90dc240fb59773ea